packages/components/psammead-section-label/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.3 | [PR#3133](https://github.com/bbc/psammead/pull/3133) Change background colour from white to ghost |
```

Comment:
This will be pulled into front pages automatically and lead to ghost-coloured section labels on these front pages e.g. `https://www.bbc.com/pidgin`. 
I'd suggest making this PR have a major version bump to 5.0.0 
It'd be good to pass in the background colour as a prop e.g. `backgroundColour`, so we can use this section label on multiple different pages (I know in the near future we'll have a dark mode for some pages, so adding this as a prop would help for that, too.)

Reply:
Do we want Ghost coloured section labels?

Reply:
@greenc05 Yes we do since the Front page, STY, PGL and MAP pages will all be made to be ghost coloured.

Reply:
I was thinking 'section label' was something different! Duh